hmda/src/test/scala/hmda/api/http/admin/InstitutionAdminHttpApiSpec.scala

```diff
@@ -181,15 +164,10 @@ class InstitutionAdminHttpApiSpec extends AkkaCassandraPersistenceSpec with Must
     }
 
     "Create institution when it doesn't exist for the lei" in {
-      val testSampleInstitution =
-        institutionGen.sample
-          .getOrElse(Institution.empty)
-          .copy(LEI = lei)
-          .copy(taxId = Option("12-3456789"))
-          .copy(activityYear = 2019)
-      Put("/institutions", testSampleInstitution) ~> institutionAdminRoutes(oAuth2Authorization) ~> check {
+      val sampleEmptyInstitution = sampleInstitution.copy(activityYear = 2019)
```

Comment:
Nit: Maybe rename to `sampleNewInstitution`? Empty doesn't make a lot of sense to me

Reply:
Thats true, change it